src/main/java/fr/openent/competences/helpers/NoteControllerHelper.java

```diff
@@ -0,0 +1,32 @@
+package fr.openent.competences.helpers;
+
+import fr.openent.competences.constants.Field;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+
+public class NoteControllerHelper {
```

Comment:
Je te conseil de mettre un constructeur pour éviter à ce qu'il soit instancier comme tu as prévu de l'utiliser avec des méthodes static : 

```suggestion
public class NoteControllerHelper {


 private NoteControllerHelper() {
        throw new IllegalStateException("NoteControllerHelper class");
    }
```

sonar :  https://rules.sonarsource.com/java/RSPEC-1118